### PR TITLE
add "expand/collapse all" buttons to summary_report

### LINKED ISSUE #160: Summary Report (html) should include an "expand all" option
When the the number of SVs isn't too overwhelming, it can be useful to expand everything and scroll through the html while validating.

Example: https://storage.cloud.google.com/unresolved_mcf/us_census/pep/population_estimate_by_race/us_pep_county_after_2000_data/summary_report.html

### PR DESCRIPTION
**Pull request by enjoythecode**
Closes #160 

---

add buttons under the "Series Summaries for Sample Places" header that
expand or collapse all `<details>` tags for sample places.

uses a javascript function that queries the page for `<details>` tags, and
sets the ".open" boolean attribute on all of them.

### CODE REVIEW

**Comment by enjoythecode** [comment]
Live demo: https://2hcfim.csb.app/#places

File: util/src/main/resources/SummaryReport.ftl [hunk 1]
```diff
@@ -236,6 +236,8 @@
         <h2>
           <a name="places" href="#places">Series Summaries for Sample Places</a>
         </h2>
+        <button onclick="open_all_details()">Expand All</button>
+        <button onclick="close_all_details()">Collapse All</button>
```
Comment (pradh) [inline util/src/main/resources/SummaryReport.ftl:240]: Wonder about putting it under TOC? And have that apply to everything under TOC and later sections.

**Review by pradh — COMMENTED** [review]
Thanks for the quick fix!

**Comment by enjoythecode** [comment]
demo updated with latest revision: https://2hcfim.csb.app/#places

**Review by pradh — APPROVED** [review]
Nice!

**Pull request merged**